tests/security/apparmor/aa_enforce.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
```

Comment:
I assume you still want to use BSD style in both files

Reply:
Yes. Currently I can not see any reason to must use BSD. There are many cases here are GPLs, I just copy the header from them. Even openQA, os-autoinst and AppArmor themselves are GPLs. If we want to do that, I suggest to open a separate ticket to clarify reasons and then change script headers for a unified license.

Reply:
Ah, OK. Then keep it